Networking/Networking/Model/Refund/Refund.swift

```diff
@@ -0,0 +1,168 @@
+import Foundation
+
+
+/// Represents a decoded Refund entity.
+///
+public struct Refund: Codable {
+    public let refundID: Int
+    public let orderID: Int
+    public let siteID: Int
+    public let dateCreated: Date // gmt
+    public let amount: String
+    public let reason: String
+    public let refundedByUserID: Int
+
+    /// If true, the automatic refund is used.
+    /// When false, manual refund process is used.
+    ///
+    public let isAutomatedRefund: Bool?
+
+    /// If true, the automated refund for a payment gateway is used.
+    /// When false, the manual refund process is used.
+    ///
+    public let createAutomatedRefund: Bool?
```

Comment:
just wondering, how is this property different from `isAutomatedRefund` above? the comments seem to look pretty similar?

Reply:
So `isAutomatedRefund` is a read-only value that we receive from a refund response. It tells us that the payment gateway automatically refunded the user. `createAutomatedRefund` is a write-only value that we send in a refund request. It does not guarantee that an automated refund occurred through the payment gateway. It's a request to try and do an automated refund through the payment gateway.

Reply:
I see, thanks for explaining!